### PR TITLE
[Merged by Bors] - feat: port Analysis.NormedSpace.Basic

### PR DESCRIPTION
**Pull request by Parcly-Taxel**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by mattrobball** [comment]
Continuity is a still a little fiddly. Further diagnostics are a good idea before merging

File: Mathlib/Analysis/NormedSpace/Basic.lean
Comment (fpvandoorn) [inline Mathlib/Analysis/NormedSpace/Basic.lean:267]: remove comment?

**Review by fpvandoorn — APPROVED** [review]
LGTM, two minor comments

bors d+

**Comment by Parcly-Taxel** [comment]
bors merge

**Comment by kmill** [comment]
The maintainers became aware of the personal (and hence non-professional) nature of the commit message inserted after the PR was delegated, and I am letting you know that we decided to force push to the master branch to remove it. This unfortunately disrupts a number of open PRs and complicates the porting process.

**Pull request closed without merging**
